groovy-mode.el

```diff
@@ -147,29 +123,23 @@ The function name is the second group in the regexp.")
      (group (regexp ,groovy-symbol-regexp))))
   "Matches interface names in groovy code.")
 
+(defvar groovy-annotation-regexp
+  (rx "@" symbol-start (+ (or (syntax word) (syntax symbol))) symbol-end)
+  "Match annotation names.")
+
+
 (defvar groovy-imenu-regexp
-  (list (list "Functions" groovy-function-regexp 2)
+  (list ;;(list "Functions" groovy-function-regexp 2)
```

Comment:
Does this no longer work? Why have you commented it out?

Reply:
The function regexp was replaced with a scanning function. I'm not sure if imenu supports that?